src/processes/follow-topic-process.js

```diff
@@ -92,9 +94,49 @@ const sendMessageAsAnonymous = async (serverClient, communityMessageFormat, data
   }
   // }
 };
+
+const sendSignedMessage = async ({
+  serverChannel,
+  targetUserId,
+  targetUsername,
+  communityMessageFormatUserId,
+  text
+}) => {
+  const message = await serverChannel?.sendMessage(
+    {
+      text,
+      user_id: communityMessageFormatUserId,
+      is_auto_message: true
+    },
+    {skip_push: true}
+  );
+
+  const notificationPayload = {
+    title: targetUsername,
+    body: `${text?.substring(0, 100)}`
+  };
+
+  const dataPayload = {
+    type: 'message.new',
+    channel_id: serverChannel?.id,
+    message: text,
+    message_schema: 'text',
+    attachment: '',
+    created_at: message?.message?.created_at || new Date().toISOString(),
+    is_annoymous: 'false',
+    priority: 'high',
+    content_available: 'true'
+  };
+
+  sendChatNotificationByChannelMembers([targetUserId], {
+    notification: notificationPayload,
+    data: dataPayload
+  });
+};
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `sendSignedMessage` function is missing error handling. If an error occurs during the execution of this function, it could cause the entire process to fail. Consider wrapping the contents of this function in a try-catch block to handle any potential errors.
```diff
+  try {
    const message = await serverChannel?.sendMessage(
      {
        text,
        user_id: communityMessageFormatUserId,
        is_auto_message: true
      },
      {skip_push: true}
    );

    const notificationPayload = {
      title: targetUsername,
      body: `${text?.substring(0, 100)}`
    };

    const dataPayload = {
      type: 'message.new',
      channel_id: serverChannel?.id,
      message: text,
      message_schema: 'text',
      attachment: '',
      created_at: message?.message?.created_at || new Date().toISOString(),
      is_annoymous: 'false',
      priority: 'high',
      content_available: 'true'
    };

    sendChatNotificationByChannelMembers([targetUserId], {
      notification: notificationPayload,
      data: dataPayload
    });
+  } catch (error) {
+    console.error(`Error sending signed message: ${error}`);
+  }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->